cmd/kes/config.go

```diff
@@ -104,6 +104,11 @@ type serverConfig struct {
 				} `yaml:"tls"`
 			} `yaml:"keysecure"`
 		} `yaml:"gemalto"`
+
+		GCP struct {
+			ProjectID       string `yaml:"projectid"`
+			CredentialsFile string `yaml:"credentialsfile"`
```

Comment:
I'm not in favor of requiring another file for the credentials.
The config file then is no longer self-contained. Instead, the credentials should be specified here. Eventually, we have to
"copy" the GCP auth. key structure here:
```
{
  "name": "projects/project-id/serviceAccounts/service-account-email/keys/key-id",
  "privateKeyData": "private-key",
  "validAfterTime": "date",
  "validBeforeTime": "date",
  "keyAlgorithm": "KEY_ALG_RSA_2048"
}
(See: https://cloud.google.com/iam/docs/creating-managing-service-account-keys)
I have to explore more about the general GCP auth. mechanisms to make a informed decision...
```